.github/workflows/tests.yml

```diff
@@ -162,26 +162,10 @@ jobs:
         run: ${{ matrix.run }}
 
   regression:
-    name: Send screenshots to Percy
-    runs-on: ubuntu-latest
+    name: Percy
```

Comment:
Is there any benefit to having this inside the tests workflow, rather than just having `screenshots.yml` trigger on the same `pull_request` and `push` events?

Reply:
Yeah, it gives the **Install** and **Build** steps chance to get cached

Then when we trigger the Percy workflow it can go super fast (already installed and built)

Reply:
Makes sense – but I think this might be worth adding as a comment somewhere as it's not obvious.